meerk40t/gui/propertypanels/opbranchproperties.py

```diff
@@ -38,7 +38,7 @@ def __init__(self, *args, context=None, node=None, **kwds):
                 "default": 1,
                 "type": int,
                 "conditional": (self.operation, "loop_enabled"),
-                "label": _("Loop"),
+                "label": _("Passes"),
```

Comment:
There's a use of `passes` inside the operations that usually just copies that operation. Which actually sort of sucks since you need a discrete amount rather than an infinite amount. But, that'd also suck with regards to optimization. Loops was used here since it's the same but different than pass. Also, hatches use the term loop, but there it's shared with ezcad terminology.

I'm not sure it's wildly important but eventually calling a bunch of different things `passes` just like we call a bunch of different things `operations` is going to catch up with us.

Reply:
Reverted the change of terms back to 'loop'